src/xrEngine/Device_Initialize.cpp

```diff
@@ -50,8 +50,7 @@ void CRenderDevice::Initialize()
 
   
 #if !defined(LINUX)
-        // xxx: need to fix getting rsRGL flag, it doesn't work now
-        if (strstr(Core.Params, "-gl") || psDeviceFlags.test(rsRGL))
+        if (psDeviceFlags.test(rsRGL))
 #endif
```

Comment:
Please, remove this `#if !defined(LINUX)` block. You have fixed the problem because of which it was here =)